src/devices/cpu/gigatron/gigatron.cpp

```diff
@@ -13,7 +13,7 @@
 #include "gigatrondasm.h"
 
 
-DEFINE_DEVICE_TYPE(GTRON, gigatron_cpu_device, "gigatron_cpu", "Gigatron CPU")
+DEFINE_DEVICE_TYPE(GTRON, gigatron_cpu_device, "gigatron_cpu", "Gigatron")
```

Comment:
Please stop doing this.  “Gigatron” is the name of the CPU, the instruction set, and the computer.  It needs to say “CPU” here to be unambiguous.

Reply:
Alright, makes sense.